utils/estimated_histogram.hh

```diff
@@ -233,6 +241,8 @@ public:
         }
         _count += b._count;
         _sample_sum += b._sample_sum;
+        _min = (b._min < _min) ? b._min : _min;
+        _max = (b._max > _max) ? b._max : _max;
```

Comment:
std::min/std::max.

What if one of them had _min = 0 because there were no values?